extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -548,7 +544,7 @@ export async function convertGithubNwoToDatabaseUrl(
     };
   } catch (e) {
     void extLogger.log(`Error: ${getErrorMessage(e)}`);
-    throw new Error(`Unable to get database for '${githubRepo}'`);
+    throw new Error(`Unable to get database for '${nwo}'`);
```

Comment:
I tried to clean up the flow for this file, so that we do each operation only once. However, it means that here we return not the exact user input but our derived `nwo` in the error.